src/app/shared/error/error-retry.component.ts

```diff
@@ -0,0 +1,23 @@
+import { Component, Input } from '@angular/core';
+import { Store } from '@ngrx/store';
+import { AllActions, CmsAccountRetryActionsAction } from '../../ngrx/actions/';
+
+@Component({
+  selector: 'metrics-error-retry',
+  template: `
+    <section *ngIf="retryActions?.length">
+      <p class="error">Something went wrong with loading metrics data. Click 'Retry' to try again.</p>
+      <button class="button" (click)="retry()">Retry</button>
```

Comment:
Probably out of scope for this ... but would be nice to somehow capture global uncaught errors, somehow report them to us, and have retry() actually refresh the page.

Reply:
Ah, okay, i missed/forgot about that!  "wouldn't-it-be-nice" withdrawn.